docs/0_prerequisites.md

```diff
@@ -0,0 +1,14 @@
+# 💼 Prerequisites
+
+In order to use ReVanced Manager, certain requirements must be met.
+
+## 🤝 Requirements
+
+- An Android device running Android 8 or higher
+- Any device architecture except ARMv7
+
```

Comment:
```suggestion
- Any device architecture except ARMv7[^1]

[^1]: You can check your device's architecture by using ReVanced Manager[^2]. Navigate to the **Settings** tab from the bottom navigation bar and scroll down to **About** under the **Info** section.
[^2]: Refer to the [⬇️ Installation page](1_installation.md) on how to download ReVanced Manager.
```

Needs a way to explain to the user how to determine their devices arch. (But uh this file comes before the Installation page so I'm not sure if this is correct).

Reply:
I don't think it is appropriate to guide the user at this point on how to check the device architecture. A new page under "usage" can be added that walks the user through all settings of ReVanced Manager. That page could mention the architecture which is visible in the settings. A hyperlink can then be added to this prerequisite, that links fo that page. Alternatively a footnote can be added on this page to link to it. In any case, the way suggested right now tries to squish in a guide how to termine the architecture on a prerequisite page.

Reply:
>  Alternatively a footnote can be added on this page to link to it.
I like this suggestion best. But also a question, does the Manager give any sort of warning on first launch if it detects ARMv7?

Reply:
Before trying to patch, it does

Reply:
ReVanced Manager works properly for apps that don't require resource patches. In these cases ARMv7 is not a problem. The prerequisite isn't 100% correct as is, not sure how else to formulate

Reply:
Ah right, then the prerequisites would need some sort of footer to explain such.

Reply:
That would work, can you suggest one?

Reply:
Yes, I agree with this issue. An alternative solution would be that this is added as a note in the usage page for patching. The note can say, that on ARMv7, some patches may not work

Reply:
I've added a footnote.

Reply:
Helping the user determine their architecture is still needed.

Reply:
That wouldn't be part of this doc. This doc is specifically for ReVanced Manager, not a tutorial on how to find the architecture. Before patching, ReVanced Manager also warns about ARMv7

Reply:
Also just realized when revanced/revanced-patcher#165 gets merged soon (hopefully) this won't even matter.

Reply:
I have added a new page for settings which should be reviewed: [`eaca18d` (#911)](https://github.com/revanced/revanced-manager/pull/911/commits/eaca18df7422c898fd62b1e1aa0c4466d719ce20).